lang/wasm_alien_thread_runner.cc

```diff
@@ -10,11 +10,15 @@
 #include <seastar/core/alien.hh>
 #include <seastar/core/posix.hh>
 #include <seastar/core/reactor.hh>
+#include <unistd.h>
 
+#include "log.hh"
 #include "lang/wasm.hh"
 #include "lang/wasm_alien_thread_runner.hh"
 #include "seastar/core/posix.hh"
 
+extern logging::logger wasm_logger;
```

Comment:
`wasm_logger` is marked as `static`, so it does not have external linkage. might want to remove `static` specifier. so linker is able to find it.

Reply:
Ok